src/main/java/org/telegram/bot/engine/Scheduler.java

```diff
@@ -0,0 +1,30 @@
+package org.telegram.bot.engine;
+
+import java.util.Calendar;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class Scheduler {
+
+    public static void scheduleTaskAt(Runnable task, Calendar scheduledTime) {
+        Calendar currentTime = Calendar.getInstance();
+        if (scheduledTime.before(currentTime)) {
+            scheduledTime.add(Calendar.DAY_OF_MONTH, 1);
+        }
+        long initialDelay = scheduledTime.getTimeInMillis() - currentTime.getTimeInMillis();
+
+        ScheduledExecutorService executorService = Executors.newScheduledThreadPool(1);
+        executorService.scheduleAtFixedRate(task, initialDelay, TimeUnit.DAYS.toMillis(1), TimeUnit.MILLISECONDS);
+    }
+
```

Comment:
Utility method for convenient Calendar creation. Better solution:
Create value object DayTime with an hour and minute properties. Use it in scheduleTaskAt:
public static void scheduleTaskAt(Runnable task, DayTime scheduledTime)

My point is to restrict the import of java.util.Calendar by only that class file.